app/rankers/__init__.py

```diff
@@ -22,7 +25,10 @@ def get_all_rankers():
         top15,
         top30,
         top45,
-        thompson_sampling,
+        thompson_sampling,  # this uses a 28 day window by default
```

Comment:
Nit: I would've just added a `thompson_sampling_28day` partial function to make this explicit all the way.